src/libs/auth.js

```diff
@@ -17,24 +18,47 @@ import { asyncImportJobStore, authStore, cookieReadyStore, requesterPaysProjectS
 import * as Utils from 'src/libs/utils'
 
 
+export const getOidcConfig = () => {
+  return {
+    authority: `${getConfig().orchestrationUrlRoot}/oauth2/authorize`,
+    client_id: getConfig().oidcClientId,
+    redirect_uri: window.origin,
+    prompt: 'login',
+    scope: 'openid email profile',
+    metadata: {
+      authorization_endpoint: `${getConfig().orchestrationUrlRoot}/oauth2/authorize`,
+      token_endpoint: `${getConfig().orchestrationUrlRoot}/oauth2/token`
+    }
+  }
+}
+
 const getAuthInstance = () => {
-  return window.gapi.auth2.getAuthInstance()
+  return authStore.get().authContext
 }
 
 export const signOut = () => {
   // Note: Ideally, we'd actually clear the Leo proxy cookie here, but there's not currently an endpoint to do that.
   // When IA-2236 is done, add that call here.
   cookieReadyStore.reset()
   sessionStorage.clear()
-  getAuthInstance().signOut()
+  getAuthInstance().removeUser()
+}
+
+export const signIn = includeBillingScope => {
+  if (includeBillingScope) {
+    return getAuthInstance().signinPopup({ scope: 'openid email profile https://www.googleapis.com/auth/cloud-billing' })
```

Comment:
If you wanted to make each scope a bit more obvious, for example, you could:
```suggestion
    return getAuthInstance().signinPopup({ scope: `
      openid
      email
      profile
      https://www.googleapis.com/auth/cloud-billing
    `.trim().split(/\s+/).join(' ') })
```

Reply:
Done